notebook/templates/lab.html

```diff
@@ -6,8 +6,6 @@
 
     <title>{{page_title}}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link rel="stylesheet" type="text/css" href="{{static_url("components/font-awesome/css/font-awesome.css")}}"></link>
```

Comment:
No, because the whole thing needs to be self contained.  (I used to work in a lab with no internet access).